README.md

```diff
@@ -12,10 +12,11 @@ The program brings up the WiFi and the underlying network interface, and uses it
 
 * [UBLOX Odin board](https://developer.mbed.org/platforms/ublox-EVK-ODIN-W2/) built-in WiFi module
 * [REALTEK_RTL8195AM](https://developer.mbed.org/platforms/REALTEK-RTL8195AM/) built-in WiFi module
-* [NUCLEO_F401RE](https://developer.mbed.org/platforms/ST-Nucleo-F401RE/) with [X-NUCLEO-IDW01M1](https://developer.mbed.org/components/X-NUCLEO-IDW01M1/) WiFi expansion board using pins D8 D2
```

Comment:
Is this now different expansion board?
Do we need to order new ones in order to test this?

Reply:
Also, if its different, why do we drop the support for previous one.

Reply:
There are two WiFi boards now from ST: X-NUCLEO-IDW01M1 (using Morpho connector) and X-NUCLEO-IDW04A1 (using Arduino connector).

Reply:
Why do you believe we are dropping one?

Reply:
Because you removed the reference to `X-NUCLEO-IDW01M1 ` and added `X-NUCLEO-IDW04A1`

Please add clarification to README.md that both are supported.

Reply:
Pls. take a closer look.